Directory.Packages.props

```diff
@@ -23,6 +23,7 @@
     <PackageVersion Include="Azure.ResourceManager.ContainerService" Version="1.2.5" />
```

Comment:
Please update PR desc with all new commands.

I just merged the postgres fixes too #866

Reply:
done https://github.com/Azure/azure-mcp/pull/855/commits/dddf8c3b2967fcd30d65aee29eb4f9eba3bebdea